anomalib/data/avenue.py

```diff
@@ -172,23 +173,23 @@ class Avenue(AnomalibVideoDataModule):
         clip_length_in_frames (int, optional): Number of video frames in each clip.
         frames_between_clips (int, optional): Number of frames between each consecutive video clip.
         task TaskType): Task type, 'classification', 'detection' or 'segmentation'
-        image_size (Optional[Union[int, Tuple[int, int]]], optional): Size of the input image.
+        image_size (int | tuple[int, int] | None, optional): Size of the input image.
```

Comment:
Could you explain why we switch from `Tuple[int, int]` to `tuple[int, int]`?

Reply:
With the new annotations, there is no need for `from typing import Tuple`. We could just do `tuple[int, int]`